examples/helloworld/src/routes/crud/delete.ts

```diff
@@ -2,7 +2,7 @@ import { Route, RouteContext } from '@seagull/routes'
 import { Term } from '../../items/index'
 
 export default class extends Route {
-  static method = 'post'
+  static method = 'POST' as const
```

Comment:
because typescript 3.4 introduced the `as const` thing.
Would need static: HttpMethod = 'POST' or 'POST' as HttpMethod

this shouldnt have worked in first place